keyboards/jiran/jiran.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "rev1.h"
+
+#include "quantum.h"
+
+// Used to create a keymap using only KC_ prefixed keys
+#define LAYOUT_kc( \
```

Comment:
Layout macros that automatically prepend `KC_` are discouraged, as they do not play well with the Configurator, and make some of the more advanced keycodes such as `MO()` and `LT()` harder to use, since you need to manually define aliases beginning with `KC_`.